docs - update people (#26493)
metabase/metabase: docs/people-and-groups/managing.md

![Admin menu](images/AdminBar.png)

## Creating an account

To add a new person, click **Add person** in the upper right corner. You’ll be prompted to enter their name and email address.

If you’ve already [configured Metabase to use email](../configuring-metabase/email.md), Metabase will send the new user an invite email. Otherwise, it’ll give you a temporary password that you’ll have to send to the person you’re inviting by hand.

## Editing an account

> You can edit someone's name and email address by clicking the three dots icon and choosing **Edit Details**. Note: be careful when changing someone's email address, because _this will change the address they’ll use to log in to Metabase_.

Reviewer: ```suggestion
You can edit someone's name and email address by clicking the three dots icon and choosing **Edit Details**. 

> Be careful: changing an account's email address _will change the address the person will use to log in to Metabase_.
```

It's important that your All Users group should never have _greater_ access for an item than a group for which you're trying to restrict access — otherwise the more permissive setting will win out. See [Setting permissions](../permissions/start.md).

## Creating a group

Go to **Admin settings** > **People** > **Groups**, and click the **Add a group** button.

We recommend creating groups that correspond to the teams your company or organization has, such as Human Resources, Engineering, Finance, and so on. By default, newly created groups don’t have access to anything.

To remove a group, click the X icon to the right of a group in the list to remove it (remember, you can’t remove the special default groups).

## Adding people to groups

> Click into a group and then click `Add members` to add people to that group. Click on the X on the right side of a group member to remove them from that group. You can also add or remove people from groups from the People list using the dropdown in the Groups column.

Reviewer: ```suggestion
To add people to that group, click into a group and then click **Add members**. 

To remove someone from that group, click on the **X** to the right of the group member. 

You can also add or remove people from groups from the **People** list using the dropdown in the **Groups** column.
```
